cfpb/design-manual
# Remove Topdoc

GitHub is reporting that this repo has a vulnerability in uglify-js, which is several levels down our dependency tree. (We now use uglify-es for our actual uglification.) The root dependency that leads to uglify-js is `grunt-topdoc`, which we no longer use in this project.

## Removals

- grunt-topdoc dependency and its associated Grunt tasks and templates


## Testing

1. Pull branch
2. `rm -rf node_modules/`
3. Run `./setup.sh` and verify that everything works correctly


## Checklist
